pass/disk.go

```diff
@@ -14,46 +14,51 @@ import (
 	sfuzzy "github.com/sahilm/fuzzy"
 )
 
+// StoreDefinition defines a password store object
+type StoreDefinition struct {
```

Comment:
Could you please also adjust `pass/disk_test.go` for these changes?

Reply:
Yes. I just need to figure out what needs doing - I don't work much with Go, so I'm a bit slow with it.

Reply:
I can fix the tests myself after merging too, that's not a big deal 🙂

Reply:
That works too :-). If you're happy to do that, I would definitely prefer it - you are likely to pick up things in testing that I may miss due to my lack of familarity.